docs/examples/non_linear_pipeline_with_multiple_backends/README.md

```diff
@@ -0,0 +1,81 @@
+# Stable diffusion
+
+This example focuses on showcasing two of Triton Inference Server's features:
+* Using multiple frameworks in the same inference pipeline
+* Using the Python Backend API to build complex non linear pipelines.
+
+## How to run?
+
+Before starting, clone this repository and navigate to the root folder. It is recommended to use three different terminals for ease.
+
+### Step 1: Prepare the Server Environment
+* First, run the Triton Inference Server Container.
+```
+docker run --gpus=all -it --shm-size=256m --rm -p8000:8000 -p8001:8001 -p8002:8002 -v ${PWD}:/workspace/ -v ${PWD}/model_repository:/models nvcr.io/nvidia/tritonserver:22.08-py3 bash
+```
+* Next, we need to install all the dependencies required by the models running in the python backend.
+
+```
+# PyTorch & Transformers Lib
+pip install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu116
+pip install transformers ftfy scipy
+pip install transformers[onnxruntime]
+```
+
+As of the creation of this example, there are some unmerged optimizations for the diffusers repository, which are being used to accelerate the UNet Model. If you DO NOT wish to use the optimizations:
+```
+pip install diffusers
+```
+If you wish to use the optimizations:
+```
+# this pip install take ~ 20 minutes
+pip install git+https://github.com/facebookresearch/xformers@51dd119#egg=xformers
+git clone https://github.com/MatthieuTPHR/diffusers.git
+cd diffusers
+git checkout memory_efficient_attention
+pip install -e .
+export USE_MEMORY_EFFICIENT_ATTENTION=1
+```
+
+### Step 2: Exporting and converting the models
+Use the NGC PyTorch container, to export and convert the models.
+
+```
+docker run -it --gpus all -p 8888:8888 -v ${PWD}:/mount nvcr.io/nvidia/pytorch:22.08-py3
```

Comment:
We need to avoid hardcoding release branches. This requires our poor DevOps Engineer to update this each release. Use `xx.yy`. This is common nomenclature throughout the entire repo, but you could make an mention of it.